includes/class-wc-update-client.php

```diff
@@ -169,9 +169,9 @@ public function print_admin_scripts() {
 		// Only need this JS/CSS on the plugin admin page and updates page.
 		if ($screen->base === 'plugins' || $screen->base === 'plugin-install') {
 			// This will make the jQuery below work with various languages.
-			$text1 = esc_html__('Compatible up to:');
-			$text2 = esc_html__('Reviews');
-			$text3 = esc_html__('Read all reviews');
+			$text1 = esc_html__('Compatible up to:', 'codepotent-update-manager');
+			$text2 = esc_html__('Reviews', 'codepotent-update-manager');
+			$text3 = esc_html__('Read all reviews', 'codepotent-update-manager');
```

Comment:
This looks like a partial update to the Update Manager code. I'm not sure this change is correct anyway but that would be an issue to take up with https://github.com/codepotent/Update-Manager.

Since this is not related to the purpose of this PR it should be removed from here and written up separately.